INSTALL.doc

```diff
@@ -34,17 +31,25 @@ To produce all the documents, the following tools are needed:
   - Antlr4 runtime for Python 3
 
 
-Under Debian based operating systems (Debian, Ubuntu, ...) a
-working set of packages for compiling the documentation for Coq is:
+Under recent Debian based operating systems (Debian 10 "Buster",
+Ubuntu 18.04, ...) a working set of packages for compiling the
+documentation for Coq is:
 
-  texlive texlive-latex-extra texlive-math-extra texlive-fonts-extra
-  texlive-humanities texlive-pictures latex-xcolor hevea transfig
-  imagemagick
-  python3 python-pip3
+  texlive-latex-extra texlive-fonts-recommended hevea python3-sphinx
+  python3-pexpect python3-sphinx-rtd-theme python3-bs4
+  python3-sphinxcontrib.bibtex python3-pip
 
-To install the Python packages required to build the user manual, run:
-  pip3 install sphinx sphinx_rtd_theme beautifulsoup4 antlr4-python3-runtime pexpect sphinxcontrib-bibtex
+Then, install the Python3 Antlr4 package:
+
+  pip3 install antlr4-python3-runtime
+
+Nix users should get the correct development enviroment from the main
```

Comment:
typo

Reply:
Point taken, I will open a different issue for removing LaTeX from Coq.

Reply:
See #7390